prow/config.yaml

```diff
@@ -2392,6 +2392,53 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- name: ci-kubernetes-e2e-gce-1-6-1-7-master-skew
+  interval: 2h
+  agent: kubernetes
+  always_run: false
```

Comment:
`always_run` doesn't really make sense to set on a CI job

Reply:
My original intention:
Don't block pipeline if this newly added test fails. When we have enough confidence that this test is doing the right thing and is it on par with ci-kubernetes-e2e-gce-1-6-1-7-upgrade-master, treat it as any other ci test.

How to achieve that?

Reply:
Changed it to always_run:true

Reply:
Removed it as per @krzyzacy 's suggestion.